util/dvsim/SimCfg.py

```diff
@@ -392,8 +392,11 @@ def prune_items(items, marked_items):
             if self.reseed_ovrd is not None:
                 test.reseed = self.reseed_ovrd
 
-            # Apply reseed multiplier if set on the command line.
-            test.reseed *= self.reseed_multiplier
+            # Apply reseed multiplier if set on the command line. This is
+            # always positive but might not be an integer. Round to nearest,
+            # but pick at least one.
```

Comment:
```suggestion
            # but pick at least 1.
```

(I read it initially as "pick at least one multiplier" or something like that.)

Reply:
Oh! That was super-unclear! Fixed.